src/functions-and-arrays.js

```diff
@@ -1,16 +1,51 @@
 // Iteration #1: Find the maximum
+const maxOfTwoNumbers = (num1, num2) => (num1 > num2 ? num1 : num2);
+
+maxOfTwoNumbers(2, 4);
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
+function findLongestWord(words) {
+let longest = '';
+for (let i = 0; i < words.length; i++) {
+if (words[i].length > longest.length)
+longest = words[i];
+}
+return longest;
+}
+
+console.log(findLongestWord(words));
```

Comment:
It seems the Jasmine tests for this function are failing with the following warning 👉🏾  `Find the longest word should return null when called with an empty array`

The test expects there to be some handling for empty arrays. Specifically, it wants the function to return null if an empty array is passed.

---

One thing you can do here is to add the empty array handling before any of the function's operations. This way, you ensure that you are taking care of this edge case before anything else.

```suggestion
function findLongestWord(words) {
/* Handle empty array edge case here:*/
 // 1. Check if the incoming array is empty.
 // 2. Return "null" if it is.
 
// code goes here...
/* Function's operation if the incoming array is not empty */ 

let longest = '';
for (let i = 0; i < words.length; i++) {
if (words[i].length > longest.length)
longest = words[i];
}
return longest;
}

console.log(findLongestWord(words));
```